doc/source/tutorials/tiling-sparse.rst

```diff
@@ -30,8 +30,9 @@ Basic concepts and definitions
 
     This is the number of non-empty cells stored in a data tile. We will
     explain soon that all data tiles in a sparse array have the same capacity,
-    which is specified by the user upon the array creation.
-
+    which is specified by the user upon the array creation. Note that **tile
```

Comment:
This is not accurate - please see below.

Reply:
reverted